holoviews/plotting/bokeh/callbacks.py

```diff
@@ -152,39 +164,84 @@ def __init__(self, plot, streams, source, **params):
         self.streams = streams
         self.comm = self._comm_type(plot, on_msg=self.on_msg)
         self.source = source
+        self.handle_ids = defaultdict(list)
 
 
     def initialize(self):
         plots = [self.plot]
         if self.plot.subplots:
             plots += list(self.plot.subplots.values())
 
```

Comment:
I see, I was looking for ``self.handle_ids`` but ``cb`` here is another ``Callback`` instance. In that case, how about merging to a different place e.g ``cb.joint_handle_ids``? If it is too complicated, then I don't mind leaving it as it is.